src/main/resources/org/datadog/jmxfetch/default-jmx-metrics.yaml

```diff
@@ -0,0 +1,145 @@
+# Memory
```

Comment:
I see multiple differences between `default-jmx-metrics.yaml` and the default metrics that we used to collect in `jmx-1` and `jmx-2` . It's very important that we don't remove/rename any metrics as it would break the graphs, monitors, etc built around them. 

Here are the diffs that I found Before => Now :
- `jvm.thread_count` => `thread.count`
- `jvm.gc.parnew.time` => `jvm.gc.minor_collection_time` ?
- `jvm.gc.cms.count` => `jvm.gc.minor_collection_count` ?

There are also multiple new metrics introduced, do we need to add them and would it be possible to move them to another PR ? That way we can focus on keeping compatibility. Improving the testing here might make sense.

Reply:
I've pushed that change - please let me know if this is an acceptable solution.

Reply:
Thanks for that. This sounds acceptable and adding 4 new metrics should not be an issue. This would also have the nice side effect of allowing us to remove some complexity from the config files of other jmx integrations like cassandra.
We should remove `jvm.gc.parnew.time` and `jvm.gc.cms.count` from the docs and add the new ones after this is merged.

Reply:
I'll discuss this changes with my team to see if there is anything I am missing on why this was not added previously. I'll then do some sanity check with A5 and A6 and if everything looks good I'll merge